src/_includes/layouts/base.njk

```diff
@@ -5,11 +5,92 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>{{ title }}</title>
   {% include "partials/global/stylesheets.njk" %}
+  {% include "partials/global/scripts.njk" %}
 </head>
 <body>
-    {% include "partials/global/language-selector.njk" %}
-    <h1>{{ title }}</h1>
-    {{ content | safe }}
+    {% include "partials/global/header.njk" %}
+    {% include "partials/components/navigation.njk" %}
+    {# example datas TODO: populate datas from different collections #}
+    {% set displayItems = ['1', '2', '3'] %}
+    {% set cardTitle = "Card Title" %}
+    {% set cardBody = "Lorem ipsum dolor sit amet, consectetur adipiscing elit" %}
+    {% if hasBanner %}
+    <div class="banner bg-{{ bannerBGColor }}">
+      <div class="banner__text">
+        <div class="banner__title">
+          {{ bannerTitle | safe }}
+        </div>
+        {{ bannerBody }}
+      </div>
+      {% if bannerImage %}
+      <div class="banner__image">
+        {{ bannerImage }}
+      </div>
+      {% endif %}
+    </div>
+    {% endif %}
+    {% block content %}{% endblock %}
+    {% if displayProjects %}
+    {% set cardImage = "yes" %}
+    <section class="display">
+      <div class="display__title">
+        <h2>Projects</h2>
+      </div>
+      <div class="display__items">
+        {% set cardType = "project" %}
+        {% for item in displayItems %}
+          {% include "partials/components/card.njk" %}
```

Comment:
I think it would be a good idea to set up these cards so that they draw from actual content and don't display unless the content exists. Right now this PR adds boilerplate but could be improved by removing that boilerplate in favour of actual functionality which will start working as soon as we populate the site with content (e.g. projects).